### PR TITLE
Start getting ember-freestyle set up for boxel-ui

### PR DESCRIPTION
**Pull request by lukemelia**
The freestyle app doesn't yet include the component CSS. We can fix this once our CSS addon is ready. 

https://user-images.githubusercontent.com/353/233208195-41939ac8-9ab7-477c-917e-e3850946d69b.mov

### CODE REVIEW

File: packages/boxel-ui/.ember-cli.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+setEdition('octane');
+
+module.exports = {
+  /**
+    Ember CLI sends analytics information by default. The data is completely
+    anonymous, but there are times when you might want to disable this behavior.
+
+    Setting `disableAnalytics` to true will prevent any data from being sent.
+  */
+  port: 4210,
```
Comment (backspace) [inline packages/boxel-ui/.ember-cli.js:17]: I really appreciate a default vs me having to remember my own scheme for our various Ember applications 🎉

**Review by backspace — CHANGES_REQUESTED** [review]
This looks good to me although I got this error when trying to start it:

```
Directory not found: /Users/b/Documents/Cardstack/Code/boxel-motion/packages/boxel-ui/tests/dummy/app/styles
```

I created an empty file at `packages/boxel-ui/tests/dummy/app/styles/app.css` and then it started up successfully.

**Comment by lukemelia** [comment]
@backspace I fixed the missing directory issue, and added test coverage that would catch this in CI.

**Review by backspace — APPROVED** [review]
As an aside, should this be v2-ified? Does that matter in a monorepo?

**Review by burieberry — APPROVED** [review]
Updated versions of the css files are in host's app.css, but we can move them out of there after the merge. I'm open to flattening the component directory structure so that all the component files aren't named `index.gts`or `usage.gts`

**Review by habdelra — APPROVED** [review]

**Pull request merged**